src/sentry/integrations/jira/integration.py

```diff
@@ -283,6 +283,10 @@ def get_client(self):
             self.model.metadata["base_url"],
             JiraCloud(self.model.metadata["shared_secret"]),
             verify_ssl=True,
+            logging_context={
+                "org_id": self.organization_id,
+                "integration_id": self._org_integration.integration.id,
```

Comment:
@ceorourke Are you 100% sure that we always will have `self._org_integration. integration ` available? I am concerned this might blow up if that evaluates to `None`. Might be worth writing in a way where we protect against that possibility.

Reply:
Also, I think we should pass in both the `integration_id` and the id of the org integration (`self._org_integration.id`) since we should have both and they are actually different things.

Reply:
ended up using `attrgetter` as suggested by Dan :)